mmda/predictors/lp_predictors.py

```diff
@@ -1,11 +1,12 @@
-from typing import Union, List, Dict, Any, Optional
+from typing import List, Dict, Optional
 
+from PIL.Image import Image
```

Comment:
explicitly importing `PIL` instead of type annotations via `PIL.Image` might cause import errors if `layoutparser` dependencies are not installed. Please check on a minimal installation.